easyrest/models/user.py

```diff
@@ -10,15 +8,19 @@
     ForeignKey,
 )
 from sqlalchemy.orm import relationship
+from passlib.hash import pbkdf2_sha256
+from validator import validation
```

Comment:
Pls move custom module import acording to pep8(below .meta)

Reply:
No problem